### PR TITLE
more readable

### PR DESCRIPTION
**Pull request by dvoet**
Ticket: <Link to Jira ticket>
<Put notes here to help reviewer understand this PR>

---

**PR checklist**

- [ ] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dsde/workbench/sam/service/PolicyEvaluatorService.scala [hunk 1]
```diff
@@ -72,35 +72,36 @@ class PolicyEvaluatorService(
...
+            hasAction && authDomainOK
+          }
+      }
+    } yield result.getOrElse(false)
+  }
 
-          // if the resource type OR action are NOT auth domain constrainable... just return
-          if (!rt.isAuthDomainConstrainable || !rt.actionPatterns.exists( ap => ap.authDomainConstrainable && ap.matches(action)) ) {
-            hasAction
+  private def isSingleAuthDomainMember(resource: FullyQualifiedResourceId, userId: WorkbenchUserId): IO[Boolean] = {
```
Comment (kcibul) [inline src/main/scala/org/broadinstitute/dsde/workbench/sam/service/PolicyEvaluatorService.scala]: thinking out loud... could there be a better name?  It's also true if this resource isn't actually constrained (regardless of if you're a member).  maybe isAuthDomainOkForSingleAuthDomainMember.... bleh.   Don't have a great suggestion for you

**Review by kcibul — APPROVED** [review]
assuming tests pass, nit on naming with no great suggestions

**Comment by dvoet** [comment]
jenkins retest

**Comment by dvoet** [comment]
jenkins retest

**Pull request closed without merging**